moto/secretsmanager/models.py

```diff
@@ -61,6 +61,7 @@ def __init__(
         secret_binary=None,
         description=None,
         tags=[],
+        KmsKeyId=None,
```

Comment:
Variables should be snake-case (`kms_key_id`) (everywhere)

Reply:
Addressed, does a module somewhere else address that the boto API has the values in camel-case?

Reply:
Ah, I missed that - no, we'd have to update the `response.py` file as well to read that param:
https://github.com/spulec/moto/blob/master/moto/secretsmanager/responses.py#L43

That goes for both create_ and update_secret.

Reply:
Got the unit tests running locally, so have fixed that now. Thanks for the help btw